### ISSUE TITLE
fix: texts are with wrong color on right side in split slide (dark mode)

### ISSUE THREAD

**Issue by afucher** [mkobuolys/flutter_deck#12]
**Description**
When using Split slides, the text in the right side in dark mode is really hard to see, the color is to brightness for the background color. 

**Steps To Reproduce**
This code is a slide where I can reproduce:
```dart
import 'package:flutter/material.dart';
import 'package:flutter_deck/flutter_deck.dart';

class Renderer extends FlutterDeckSlideWidget {
  const Renderer()
      : super(
          configuration: const FlutterDeckSlideConfiguration(
            route: '/renderer',
            header: FlutterDeckHeaderConfiguration(
              title: 'Web renderers',
            ),
          ),
        );

  @override
  FlutterDeckSlide build(BuildContext context) {
    return FlutterDeckSlide.split(
      leftBuilder: (context) {
        return const Text('HTML renderer');
      },
      rightBuilder: (context) {
        return const Text('CanvasKit renderer');
      },
    );
  }
}

```
**Expected Behavior**

The text in the right side should be darker.

**Screenshots**
<img width="862" alt="Captura de Tela 2023-09-17 às 02 10 10" src="https://github.com/mkobuolys/flutter_deck/assets/3756185/baeaf782-d2b2-4e3f-93d5-9843c7152cdf">

**Additional Context**
I running on a MacOS and Chrome.

**Comment by afucher** [comment]
The same slide in the light mode to comparison
<img width="813" alt="Captura de Tela 2023-09-17 às 02 34 48" src="https://github.com/mkobuolys/flutter_deck/assets/3756185/ca9fc2a5-28f1-4dd9-a67c-2724a445740c">

**Comment by mkobuolys** [comment]
Hi @afucher, thanks for your input. This bug occurs because you are using `Text` without applying any style property. By default, the light theme uses black text and white for the dark one. When creating slides, you are expected to use the `FlutterDeckTheme` so that your text is styled properly. E.g:

```
import 'package:flutter/material.dart';
import 'package:flutter_deck/flutter_deck.dart';

class Renderer extends FlutterDeckSlideWidget {
  const Renderer()
      : super(
          configuration: const FlutterDeckSlideConfiguration(
            route: '/renderer',
            header: FlutterDeckHeaderConfiguration(
              title: 'Web renderers',
            ),
          ),
        );

  @override
  FlutterDeckSlide build(BuildContext context) {
    return FlutterDeckSlide.split(
      leftBuilder: (context) {
        return Text(
          'HTML renderer',
          style: FlutterDeckTheme.of(context).textTheme.bodyLarge, // Using one of the FlutterDeck text styles
        );
      },
      rightBuilder: (context) {
        return Text(
          'CanvasKit renderer',
          style: FlutterDeckTheme.of(context).textTheme.bodyLarge, // Using one of the FlutterDeck text styles
        );
      },
    );
  }
}
```

Or you can use the global Material theme of the app (updated with https://github.com/mkobuolys/flutter_deck/pull/13, released with v0.4.1):

```
import 'package:flutter/material.dart';
import 'package:flutter_deck/flutter_deck.dart';

class Renderer extends FlutterDeckSlideWidget {
  const Renderer()
      : super(
          configuration: const FlutterDeckSlideConfiguration(
            route: '/renderer',
            header: FlutterDeckHeaderConfiguration(
              title: 'Web renderers',
            ),
          ),
        );

  @override
  FlutterDeckSlide build(BuildContext context) {
    return FlutterDeckSlide.split(
      leftBuilder: (context) {
        return Text(
          'HTML renderer',
          style: Theme.of(context).textTheme.bodyLarge, // Using one of the Material theme text styles
        );
      },
      rightBuilder: (context) {
        return Text(
          'CanvasKit renderer',
          style: Theme.of(context).textTheme.bodyLarge, // Using one of the Material theme text styles
        );
      },
    );
  }
}
```

Still, you need to use styling for your text. Otherwise, the context does not know that a specific colour other than the default one should be applied here.

Let me know if it's clear now and I can close the issue.

**Comment by afucher** [comment]
thnks for the explanation @mkobuolys !
So, I'll suggest to change the code example for Split slide in the [readme](https://github.com/mkobuolys/flutter_deck#split-slide) to reflect this. Because there it is just using texts without the style, so in my mind it would work.

**Comment by mkobuolys** [comment]
Fair. Updated with https://github.com/mkobuolys/flutter_deck/pull/14, released with v0.4.2.
